Modules/install-jaspBase.R.in

```diff
@@ -54,5 +55,5 @@ if ("jaspBase" %in% installed.packages()) {
 # Todo, I can do this using CMake like I do on Windows
 if (Sys.info()["sysname"] == "Darwin") {
   source('@MODULES_BINARY_PATH@/symlinkTools.R')
-  convertAbsoluteSymlinksToRelative('@MODULES_BINARY_PATH@', '@MODULES_RENV_CACHE_PATH@')
+  convertAbsoluteSymlinksToRelative('@R_LIBRARY_PATH@', '@MODULES_RENV_CACHE_PATH@')
```

Comment:
@boutinb this change is necessary because we are not installing `jaspBase` to the `Modules` folder, so if we are going to run the symlink conversion we better tell it the real location. But if it works, is for you to test out ;)